src/VarnishServiceProvider.php

```diff
@@ -10,9 +10,13 @@ class VarnishServiceProvider extends ServiceProvider
     public function boot()
     {
         if ($this->app->runningInConsole()) {
-            $this->publishes([
-                __DIR__.'/../config/varnish.php' => config_path('varnish.php'),
-            ], 'config');
+            $configPath = __DIR__ . '/../config/varnish.php';
+            if (function_exists('config_path')) {
+                $publishPath = config_path('varnish.php');
+            } else {
+                $publishPath = base_path('config/varnish.php');
```

Comment:
We only need this live in order to make it work in both Lumen and Laravel right?

Reply:
Yes. I just tested with lumen 5.6 (nginx 1.14 + varnish 5.2 + php-fpm 7.2) with suggested varnish vcl.

P.S.
I presume lumen users already know that there is no auto discovery like in Laravel 5.6 and they should load the config file before route middleware definition like below but you may want to add these details into readme file.

'$app->configure('varnish');
$app->routeMiddleware([
'cacheable' => \Spatie\Varnish\Middleware\CacheWithVarnish::class,
]);'

Reply:
Please add instruction to the readme in this PR?

Could you also remove that if condition. Just keep line 17.